### ISSUE TITLE
LevelZ Reseting Attack on Server

### ISSUE THREAD

**Issue by RazarJP** [Globox1997/LevelZ#46]
I'm running LevelZ on my home server, I'm playing with my friends.

I've noticed that when we reconnect on game, the attack bonus just vanish, it reset, points still.
Def Bonus and others are normals and keep it after reconnect.

Running same config on single player it don't happen.

**Comment by Limesar** [comment]
I've been experiencing this issue in my singleplayer world too, even on the latest version. Like Razar mentioned, this only affects the attack stat for whatever reason.

What's strange is that, when I added an unrelated mod to my mods folder, my attack stat _was_ properly initialized... only to revert to the base stat upon exiting and re-entering the world just like every past instance. I seriously have no idea why adding (or maybe removing) a different mod would change things for even a brief moment.

Hoping this gets fixed soon - if I can play my world without having to go to the config just to edit the base attack stat every time I give it another bonus point, it'll be too soon.

**Comment by Globox1997** [comment]
[levelz-1.1.2-testone.zip](https://github.com/Globox1997/LevelZ/files/7697383/levelz-1.1.2-testone.zip)
Please test this first and then this
[levelz-1.1.2-testtwo.zip](https://github.com/Globox1997/LevelZ/files/7697384/levelz-1.1.2-testtwo.zip)

**Comment by Limesar** [comment]
Alright, I just tested both of those versions.

Test One fixed the problem of attack bonuses not persisting, but only if I saved & quit and kept Minecraft running. Closing Minecraft and restarting it still failed to apply the bonuses.

Test Two fixed the issue in both situations. I also checked to make sure none of the other stat bonuses broke in the process (mostly just a quick look at the stats shown in the GUI, might want to test further with trading/alchemy/farming/mining just to make sure), and the others don't seem to have broken.

Thank you for the fix!
